runtime/doc/lsp.txt

```diff
@@ -177,8 +177,10 @@ autocmd CursorHold *
 The following requests have built-in callbacks defined to handle the response
 in an idiomatic way.
 
-  textDocument/definition~
+  textDocument/completion~
   textDocument/hover~
+  textDocument/signatureHelp~
+  textDocument/definition~
```

Comment:
the tilde `~` could have a space before it, so that `*` or `yiw` doesn't include it if user wants to search or yank this word. Though as a side-note, I don't see why `~` highlighting is wanted here.